deepinv/loss/metric/perceptual.py

```diff
@@ -35,7 +35,7 @@ class LPIPS(Metric):
     def __init__(self, device="cpu", **kwargs):
         super().__init__(**kwargs)
         pyiqa = import_pyiqa()
-        self.lpips = pyiqa.create_metric("lpips").to(device)
+        self.lpips = pyiqa.create_metric("lpips", check_input_range=False).to(device)
```

Comment:
The pyiqa doc is not clear on that point but it seems that what you suggest here is departing from the default behaviour... How about adding `check_input_range` as an init optional parameter?

```python
     def __init__(self, device="cpu", check_input_range=False, **kwargs)
```